### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by jcorteza**

### CODE REVIEW

File: src/components/HomeTimelineUIContainer.jsx [hunk 1]
```diff
@@ -1,59 +1,114 @@
...
+    }
+
+    handleClick(e) {
+        e.preventDefault();
+        this.setState({
+            data: [],
+            fetchComplete: false,
+            errorOccurred: false
+        });
+        this.fetchData(this.state.filtered);
```
Comment (kev-chan) [inline src/components/HomeTimelineUIContainer.jsx]: what does the filtered state represent?

File: src/style/sass/main.scss [hunk 2]
```diff
@@ -64,7 +78,7 @@ body {
                     margin-top: 20px;
                 }
                 
-                .uiButton {
+                .uiButton, #filterButton, #filterInput {
```
Comment (kev-chan) [inline src/style/sass/main.scss:87]: why add ids to the scss?

**Review by kev-chan — CHANGES_REQUESTED** [review]

**Comment by kev-chan** [comment]
aren't components 'js' as well?  asking in regards to directories

File: src/style/sass/main.scss (hunk 2 above)
Comment (jcorteza) [reply src/style/sass/main.scss:87]: They're unique elements. The only components with a "filtering" related function, but I wanted their padding and margin to match.

**Review by kev-chan — CHANGES_REQUESTED** [review]
please ensure you've separated code into respective areas per the instructions in the lab
- add 5 pixels left & right of the filter button

**Review by kev-chan — CHANGES_REQUESTED** [review]
clean up unused imports
- separate out utils from services as utils is nested in services when they are independent
- move index.js out since it's neither service/utils

**Comment by kev-chan** [comment]
apologies from my end, 5px  on each side of the text and not outside of the button.  (The button should be longer)

**Review by kev-chan — APPROVED** [review]

**Pull request merged**